gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/Histogram.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gobblin.salesforce;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Getter;
+
+
+@Getter
+public class Histogram {
+  private long totalRecordCount;
+  private final List<HistogramGroup> groups;
+
+  Histogram() {
```

Comment:
package-protected is quite uncommon in the project, and certainly not "the default".  if you're specifically choosing that, please explain the reasoning in the class-level javadoc.  otherwise, make this `public`, `protected`, whatever

Reply:
Updated to public for this, while keeping RecordModTimeHistogramService as it is as it contains logic too internal to this package/use case.